### PR TITLE
Add authen author to turtlebot

### PR DESCRIPTION
**Pull request by Shokman**

### CODE REVIEW

File: www/rocon_web_remocon/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 
     <!-- etc -->
-    <script type="text/javascript" src="index.js"></script>
+    <script type="text/javascript" src="/rocon_web_remocon/index.js"></script>
```
Comment (adamantivm) [inline www/rocon_web_remocon/index.html]: @Shokman can you please undo this change?

**Comment by adamantivm** [comment]
@Shokman other than that very minor issue, I think it's ready to merge. Let me know when you push that change and I'll merge.

**Comment by Shokman** [comment]
@adamantivm This and another PR's reviewed
